bindings/wysiwyg-ffi/README.md

```diff
@@ -67,13 +76,19 @@ cargo install uniffi_bindgen
 Make sure you install same version of uniffi_bindgen that is mentioned in
 `bindings/wysiwyg-ffi/Cargo.toml`, otherwise you will get weird errors.
 
-* Build shared object:
+* Build the library using Gradle:
+
+```bash
+ANDROID_SDK_ROOT=/home/andy/AndroidSdk make android
```

Comment:
I would normally set this variable as an env var in `.bashrc`, but this is probably fine.

Reply:
In the pre-requisites we're already using `ANDROID_HOME` or a similar variable, so we assume it already exists. There's probably no need to manually set it here too.

Reply:
Thanks, I updated to use ANDROID_HOME